lib/mws/off_amazon_payments/client.rb

```diff
@@ -14,14 +14,15 @@ module OffAmazonPayments
     #   Payments. You cannot use this API section to process payments for Amazon
     #   Marketplace, Amazon Webstore, or Checkout by Amazon.
     class Client < ::Peddler::Client
-      path '/OffAmazonPayments/2013-01-01/'
+      version '2013-01-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.